torch/_dynamo/optimizations/training.py

```diff
@@ -22,6 +31,62 @@
 log = logging.getLogger(__name__)
 
 
+def aot_autograd(**kwargs):
+    def compiler_fn(gm: torch.fx.GraphModule, example_inputs):
+        import functorch.compile
+
+        # Hack to get around circular import problems with aot_inductor_debug
+        if callable(kwargs.get("decompositions")):
+            kwargs["decompositions"] = kwargs["decompositions"]()
+
+        # TODO: stop monkeypatching here (without even cleaning up, UGH!)
+        functorch.compile.config.use_functionalize = True
```

Comment:
Shouldn't we just update the config's default at this point?

Reply:
Yes, see https://github.com/pytorch/pytorch/pull/89663